pennylane/ops/op_math/exp.py

```diff
@@ -180,6 +203,35 @@ def has_decomposition(self):
                 )
         return math.real(self.coeff) == 0 and qml.pauli.is_pauli_word(self.base)
 
+    @property
+    def inverse(self):
+        """Setting inverse is not defined for Exp, so the inverse is always False"""
+        return False
+
+    @inverse.setter
+    def inverse(self, boolean):
+        raise NotImplementedError(
+            f"Setting the inverse of {type(self)} is not implemented. "
+            f"Use qml.adjoint or qml.pow instead."
+        )
+
+    def inv(self):
+        """Inverts the operator.
+
+        This method concatenates a string to the name of the operation,
+        to indicate that the inverse will be used for computations.
+
+        Any subsequent call of this method will toggle between the original
+        operation and the inverse of the operation.
+
+        Returns:
+            :class:`Operator`: operation to be inverted
+        """
+        raise NotImplementedError(
+            f"Setting the inverse of {type(self)} is not implemented. "
+            f"Use qml.adjoint or qml.pow instead."
+        )
```

Comment:
They are overwritten because we don't want to inherit them - the base implementation allows you to invert, but these raise an error and tell you to use `adjoint` or `pow` if you try (and return False if you ask op.inverse).